### PR TITLE
Time based query

### PR DESCRIPTION
**Pull request by sbaskaran**
@dg0yal please review

### CODE REVIEW

**Comment by dg0yal** [comment]
Looks good. Please resolve conflicts, then this can be merged

File: arcgis-ios-sdk-samples/Features/Time based query/TimeBasedQueryVC.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        self.mapView.map = self.map
+        
+        //create feature table using a url
+        self.featureTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0")!)
+        
+        //define the request mode
+        self.featureTable.featureRequestMode = .manualCache
+        
+        //load the feature table
+        self.featureTable.load { [weak self] (error) in
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Features/Time based query/TimeBasedQueryVC.swift]: on second thoughts, lets simplify this by removing the code to load the feature table
We can just directly create layer from table and add it to map. 
populate() will implicitly load the table and if there are errors it'll get printed in the completion block.

File: arcgis-ios-sdk-samples/Features/Time based query/TimeBasedQueryVC.swift [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        self.featureTable.populateFromService(with: queryParams, clearCache: true, outFields: ["*"]) { (result:AGSFeatureQueryResult?, error:Error?) -> Void in
+            
+            //check for error
+            if let error = error {
+                SVProgressHUD.showError(withStatus: error.localizedDescription, maskType: .gradient)
+            }
+            else {
+                //the resulting features should be displayed on the map
+                //you can print the count of features
+                print(result?.featureEnumerator().allObjects.count ?? 0)
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Features/Time based query/TimeBasedQueryVC.swift]: can you add some text in this print stmt

**Comment by sbaskaran** [comment]
@dg0yal you merged/deleted before I was done pushing

**Comment by dg0yal** [comment]
i guess im trigger happy today

**Pull request merged**